.github/ISSUE_TEMPLATE/add--org--logo-to-the-website.md

```diff
@@ -45,7 +45,7 @@ You can delete the whole `### Title` section from here.
 - [ ] please edit the InnerSource in Action page located at the following path: content/community/action.md
 - [ ] please add your logo in the correct place in the list, following alphabetical order, by using the following code:
 {{< company name="organisation_name" image="/images/logos/organisation_name.png" >}}{{< /company >}}
-
+- [ ] in the Pull Request please specify why you are adding the logo to the site (E.g.: I work for the company and know they are doing InnerSource)
```

Comment:
```suggestion
- [ ] please specify in the Pull Request why you are adding the logo to the site (E.g.: I work for the company and know they are doing InnerSource)

```

Reply:
Matching sentence structure to previous bullets.
Adding spacing ahead of headline.